docs/DesignNotes.md

```diff
@@ -85,24 +85,24 @@ If there is no error, this endpoint returns:
 - the voter's ballot check
 - the voter's row offset into the ballot check
 
-### E. Fourth API endpoint (occurs during phase 3 above)
+### Fourth API endpoint (occurs during phase 3 above)
 
 - Given an connection ID, will invoke verify-ballot-receipt on the server side backend
 - Supports various switches, each switch being a different UX button
 
 Supports calling the backend python script verify-ballot-receipt with various switches. Each different python switch would map (TBD) to a different button.  It is this endpoint which is of high interest regarding the VTP demo as this endpoint basically demonstrates E2EV.  As such we probably want to plan some time optimizing the UX experience for this - the better it is, the more compelling VoteTracker+ will be for the voter.
 also requires a connection ID
 
-### F. Fifth API endpoint (occurs during phase 3 above)
+### Fifth API endpoint (occurs during phase 3 above)
 
 - Given an connection ID, will invoke tally-contests on the server side backend
 - Supports various switches, each switch being a different UX button
 
 Supports calling the backend python script tally-contests with various switches.  Each different python switch would map (TBD) to a different button.  It is this endpoint which is of high interest to the RCV folks as it is this one where the voter can see how their ranked voted is counted across the rounds of rank choice voting.  As such we probably want to plan some time optimizing the UX experience for this - the better it is, the more compelling RCV should be for the voter.
 
-## 4) Other Odds and Ends
+## Other Odds and Ends
 
-1. We decided to allow participants in the demo to vote as many times as they would like so to gain experience with RCV and VTP and to allow multiple people to use one phone.  This might be a bad decision - in retro spec it seems like it.  It may be that we want to minimize the number of things that can confuse the voter (even though in real life they will not be voting by phone).  Regardless, to prevent multiple votes the solution would need to work across multiple browsers.
+1. We decided to allow participants in the demo to vote as many times as they would like so to gain experience with RCV and VTP and to allow multiple people to use one phone.  This might be a bad decision - in retrospec it seems like it.  It may be that we want to minimize the number of things that can confuse the voter (even though in real life they will not be voting by phone).  Regardless, to prevent multiple votes the solution would need to work across multiple browsers.
```

Comment:
So close! "Retrospect" not "retrospec"

Reply:
Also, you could just delete this whole sentence. It's odd to speculate on what a bad idea a design decision is in this document. Maybe move this to Discussions, or discuss here, if you're not sure about this part of the plan?

Reply:
I am fine deleting the sentence.  The reason I was thinking it was a bad idea (to be able to vote twice) is that I can see people associating being able to vote multiple times as something inherent with VTP.  Yes we would say that it is not the case, be we are going to be saying a lot of things.  At some point too many things said will be blurred, and the blurring point is different for different folks.  But the tradeoff in this case is ok IMHO - let people vote multiple times.

Reply:
Also, I am thinking that changing and discussing the doc is ok as that is what the PR process is for.  However, I am fine if you want to bring the doc back into a discussion topic.  I was just trying it (the process) out as a checked in doc.  You can pick.